app/src/main/res/layout/activity_recommended_actions.xml

```diff
@@ -230,132 +230,6 @@
 
         </android.support.v7.widget.CardView>
 
-        <android.support.v7.widget.CardView
```

Comment:
Taking this out for the minute because there was an unmatched tag and wanted to get the PR through, I'll start another PR will cards for all tests created

Reply:
new PR up